examples/namespaced_custom_object.py

```diff
@@ -99,30 +98,28 @@ def main():
     print("Resource details:")
     pprint(resource)
 
-
-    # delete it
-    api.delete_namespaced_custom_object(
+    # patch the namespaced custom object to update the `spec.cronSpec` field
+    patch_resource = api.patch_namespaced_custom_object(
         group="stable.example.com",
         version="v1",
         name="my-new-cron-object",
         namespace="default",
         plural="crontabs",
-        body=client.V1DeleteOptions(),
+        body=patch_body,
     )
-    print("Resource deleted")
-
+    print("Resource details:")
+    pprint(patch_resource)
 
-    # patch the namespaced custom object to update the `spec.cronSpec` field
-    patch_resource = api.patch_namespaced_custom_object(
+    # delete it
+    api.delete_namespaced_custom_object(
```

Comment:
why do we have a commit that reverse the order of `patch_namespaced_custom_object` and `delete_namespaced_custom_object`?

Reply:
https://github.com/kubernetes-client/python/pull/1444/commits is what I was asking for, if that makes sense

Reply:
I kept the order like this,
we have create, get, patch, delete

Reply:
currently the two PR have the same code change, but the commits are different

Commits should represent meaningful milestones or units of work. Currently the commits in this PR are: 
- your first commit renames the file and adds a patch method after the delete method, which means the example has a bug and will run into an error
- your second commit reverses the order of the patch method and the delete method to fix the bug in the first commit

please take a look at https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md#squash-commits. The second commit is a "Work in progress" (it fixes a bug in the first commit during development), so please squash it. 

again, I think having two commits like the following will be most reasonable and readable:
- one commit for adding patch example to custom_object.py
- one commit for renaming the file to namespaced_custom_object.py

at this point I'm also okay with having just one commit, which will be more reasonable than the current two commits in this PR

Reply:
I am keeping the one commit and with proper commit message.